README.md

```diff
@@ -1,12 +1,18 @@
 
 # redux-saga-controller
 
+<p align="center"> 
```

Comment:
Can you use [markdown syntaxes](https://www.markdownguide.org/basic-syntax/) instead of HTML?

Reply:
githubs markdown doesn't support any alignment

Reply:
We don't need to support alignment, it is not so good mix HTML and markdown

Reply:
We don't need to support alignment, it is not so good mix HTML and markdown